src/apps/create-patron-user-info/UserInfo.tsx

```diff
@@ -35,27 +35,44 @@ const UserInfo: FC<UserInfoProps> = ({ cpr }) => {
     phoneNumber: "",
     emailAddress: ""
   });
-
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isSubmitError, setIsSubmitError] = useState<boolean>(false);
+  const getSubmitButtonText = () => {
+    if (isLoading) {
+      return t("createPatronButtonLoadingText");
+    }
+    if (isSubmitError) {
+      return t("createPatronButtonErrorText");
+    }
+    return t("createPatronConfirmButtonText");
+  };
```

Comment:
Could we move this to a helper.ts file? I have another PR on its way where I have introduced a helper so it could fit in there.